test/integration/test_expid.py

```diff
@@ -260,3 +279,337 @@ def test_copy_expid_no(autosubmit: Autosubmit) -> None:
     hpc_result = describe[4].lower()
 
     assert hpc_result != new_hpc
+
+
+def create_as_exp(tmp_path):
+    _user = getuser()
+
+    return {
+        'PLATFORMS': {
+            'pytest-ps': {
+                'type': 'ps',
+                'host': '127.0.0.1',
+                'user': _user,
+                'project': 'whatever',
+                'scratch': str(tmp_path / 'scratch'),
+                'DISABLE_RECOVERY_THREADS': 'True'
+            }
+        },
+        'JOBS': {
+            'debug': {
+                'SCRIPT': 'echo "Hello world"',
+                'RUNNING': 'once'
+            },
+        }
+    }
+
+
+@pytest.mark.parametrize('test, operational, evaluation, expected, new_exp', [
```

Comment:
Traditionally, params in pytest parameters are not space-separated :grimacing: I don't believe any linter checks for this...

Example: https://docs.pytest.org/en/7.1.x/example/parametrize.html#different-options-for-test-ids